exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,14 +6,16 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  return (number >= 0) ? "positive" : "negative";
```

Comment:
I can see you running with ternary operators. Really good stuff Musa

Reply:
Thank you so much Dharma😊